src/pycell.rs

```diff
@@ -8,9 +8,9 @@ use crate::type_object::{PyBorrowFlagLayout, PyLayout, PySizedLayout, PyTypeInfo
 use crate::types::PyAny;
 use crate::{ffi, IntoPy, PyErr, PyNativeType, PyObject, PyResult, Python};
 use std::cell::{Cell, UnsafeCell};
+use std::fmt;
```

Comment:
There's a bit of inconsistency, in other files, e.g. types::complex, the imports are feature-gated. Although, I think your approach here is more legible since there's just less conditionals involved.

Reply:
I agree it's not super consistent. I'm not particularly worried though, choices like this area very easy to change later :-)

FWIW, I did `std::mem::size_of` in this file specifically because there were several other existing callers that used that style in this file.

Reply:
I actually don't care about how we limit imports.
Please chooes the way you like.